R/type-rcrd.R

```diff
@@ -12,6 +12,9 @@
 #'   (same sizes), and contain uniquely named vectors (at least
 #'   one). `fields` is validated with [df_list()] to ensure uniquely
 #'   named vectors.
+#' @details
+#' A new rcrd class created with [new_rcrd()] will not have a [format()] method.
```

Comment:
Maybe that should be a more general mention that a rcrd subclass is very bare bones and requires implementing a bunch of methods to make it useful, including `format()`, and then linking to the vignette as you did here?